### ISSUE TITLE
Are we still doing the AA introduction?

### ISSUE THREAD

**Issue by jberkus** [vmbrasseur/10stepprogram#32]
Early on, we talked about doing a "funny" introduction based on AA meetings.  Are we still doing that?  I'd kind of like it to loosen people up.

**Comment by vmbrasseur** [comment]
No, I would rather not do anything jokey related to alcoholism. ;-)

What else would loosen people up?
